assets/components/productPage/productPageButton/productPageButton.jsx

```diff
@@ -4,6 +4,7 @@
 
```

Comment:
What does product page button refer to? Is that the name of the "showcase page"? In which case, maybe we should call this something more general as it's now being used on the contributions flow?

Reply:
this is why I wanna have this chat on buttons, to sort out how we wanna unify buttons. At the moment i just did this to unblock the checkout landing so we didn't have to reuse styles